packages/neuron-ui/src/components/Transaction/index.tsx

```diff
@@ -126,6 +137,50 @@ const Transaction = () => {
     )})`
   }, [transaction.outputs.length, transaction.outputsCount, t])
 
+  const onTagClicked = (lockScript: CKBComponents.Script, address: string) => {
+    setShowLockInfoDialog(true)
+    setLockInfo(lockScript)
+    setNewAddress(address)
+  }
+
+  const renderTag = (address: string) => {
+    const lockScript = addressToScript(address)
+    const newAdd = lockScriptToAddress(lockScript, isMainnet)
+    const foundLock = [
```

Comment:
The lock array is ok here, but for more precision, the lock array should be filtered by `isMainnet`.
It is possible (but almost impossible) to hit the mainnet configuration/testnet configuration on a dev chain.